arm-compute/2015-06-15/swagger/compute.json

```diff
@@ -2659,14 +2659,7 @@
       "properties": {
         "forceUpdateTag": {
           "type": "string",
-          "description": "Gets or sets how the extension handler should be forced to update even if the extension configuration has not changed.",
-          "enum": [
-            "RerunExtension"
-          ],
-          "x-ms-enum": {
-            "name": "ForceUpdateTagTypes",
-            "modelAsString": false
-          }
+          "description": "Gets or sets how the extension handler should be forced to update even if the extension configuration has not changed."
```

Comment:
The description should just be: "How the extension handler..." The individual language generators will add the Gets or Sets if it's appropriate (some languages don't support properties).  Adding Gets or Sets results in this description for C#: 
        /// Gets or sets gets or sets how the extension handler should be
        /// forced to update even if the extension configuration has not
        /// changed.